judging/requirements.txt

```diff
@@ -0,0 +1,2 @@
+# data science
```

Comment:
For now this is the only requirement (beyond Python 3 itself) that we need, but I thought it better to capture it in a `requirements.txt` anyway as we will probably add more later.